ocp_resources/aaq.py

```diff
@@ -0,0 +1,12 @@
+# API reference: https://github.com/kubevirt/application-aware-quota/blob/main/README.md
+# TODO: update API reference when OCP doc is available
+
+from ocp_resources.resource import Resource
+
+
+class AAQ(Resource):
```

Comment:
Is this resource that the user can create as well?

Reply:
I believe so - this work is going to evolve and provide more clarity in next few months. For our use case, it's deployment is currently controlled via HCO and this support was added just now.

Reply:
Please add TODO, to fill the missing code for create (to_dict())

Reply:
Added one todo.